test/test_helper.rb

```diff
@@ -58,7 +58,8 @@ def ==(other)
       # uses lots of ports on localhost for thesystem tests
       "127.0.0.1",
       "localhost:3040", "dynamodb",
-      "localhost:3041", "s3"
+      "localhost:3041", "s3",
+      "localhost:3022", "git-server"
```

Comment:
I don't think we want to require someone to have the git-server running to run the tests.

Reply:
I didn't actually need this. I initially had a bug in the stubbing of the Git Concept code. Removed.